macros/parsers/parserCheckboxList.pl

```diff
@@ -552,15 +588,38 @@ sub CHECKS {
 	if ($main::displayMode eq 'TeX') {
 		$checks[0] = "\n\\begin{itemize}\n" . $checks[0];
 		$checks[-1] .= "\n\\end{itemize}\n";
-	}
+	} elsif ($main::displayMode eq 'PTX') {
+
+		# Do we want an ol, ul, or dl?
+		my $list_type      = 'ul';
+		my $close_list     = 'ul';
+		my $originalLabels = $self->{originalLabels};
+		if ($originalLabels =~ m/^(123|abc|roman)$/i) {
+			my $marker = '';
+			$marker = "1" if $originalLabels eq '123';
+			$marker = "a" if $originalLabels eq 'abc';
+			$marker = "A"
+				if uc($originalLabels) eq 'ABC' && $originalLabels ne 'abc';
+			$marker = "i" if $originalLabels eq 'roman';
+			$marker = "I"
+				if uc($originalLabels) eq 'ROMAN' && $originalLabels ne 'roman';
```

Comment:
This is inconsistent.  Some of the `if` statements are on the same line, and some on the next.  This is a case where it would definitely improve readability to have all `if` statements on the same line.  Add an empty line after the last `$marker = ...` line if you want to prevent perltidy from trying to align those with the next `$list_type = ...` line.